plugin/miniSnip.vim

```diff
@@ -4,22 +4,23 @@
 if exists('g:loaded_miniSnip') | finish | endif
 let s:cpo_save = &cpo | set cpo&vim
 
-let g:miniSnip_dirs     = get(g:, 'miniSnip_dirs', [])
-let g:miniSnip_trigger  = get(g:, 'miniSnip_trigger', '<C-j>')
-let g:miniSnip_complkey = get(g:, 'miniSnip_complkey', '<C-x><C-u>')
-let g:miniSnip_extends  = get(g:, 'miniSnip_extends', {})
-let g:miniSnip_ext      = get(g:, 'miniSnip_ext', 'snip')
-let g:miniSnip_local    = get(g:, 'miniSnip_local', ".miniSnip")
-
-let g:miniSnip_delimChg = get(g:, 'miniSnip_delimChg', '$' )
-let g:miniSnip_descmark = get(g:, 'miniSnip_descmark', '?' )
-let g:miniSnip_opening  = get(g:, 'miniSnip_opening',  '<{')
-let g:miniSnip_closing  = get(g:, 'miniSnip_closing' , '}>')
-let g:miniSnip_refmark  = get(g:, 'miniSnip_refmark',  '~' )
-let g:miniSnip_evalmark = get(g:, 'miniSnip_evalmark', '!' )
-let g:miniSnip_finalTag = get(g:, 'miniSnip_finalTag', '+' )
-let g:miniSnip_noskip   = get(g:, 'miniSnip_noskip',   '`' )
-let g:miniSnip_named    = get(g:, 'miniSnip_named',    '@' )
+let g:miniSnip_dirs          = get(g:, 'miniSnip_dirs', [])
+let g:miniSnip_trigger       = get(g:, 'miniSnip_trigger', '<C-j>')
+let g:miniSnip_complkey      = get(g:, 'miniSnip_complkey', '<C-x><C-u>')
+let g:miniSnip_extends       = get(g:, 'miniSnip_extends', {})
+let g:miniSnip_ext           = get(g:, 'miniSnip_ext', 'snip')
+let g:miniSnip_local         = get(g:, 'miniSnip_local', ".miniSnip")
+
+let g:miniSnip_delimChg      = get(g:, 'miniSnip_delimChg', '$' )
+let g:miniSnip_descmark      = get(g:, 'miniSnip_descmark', '?' )
+let g:miniSnip_opening       = get(g:, 'miniSnip_opening',  '<{')
+let g:miniSnip_closing       = get(g:, 'miniSnip_closing' , '}>')
+let g:miniSnip_refmark       = get(g:, 'miniSnip_refmark',  '~' )
+let g:miniSnip_evalmark      = get(g:, 'miniSnip_evalmark', '!' )
+let g:miniSnip_finalTag      = get(g:, 'miniSnip_finalTag', '+' )
+let g:miniSnip_noskip        = get(g:, 'miniSnip_noskip',   '`' )
+let g:miniSnip_named         = get(g:, 'miniSnip_named',    '@' )
+let g:miniSnip_expandpattern = get(g:, 'miniSnip_expandpattern', '\f')
```

Comment:
I really don't like how long of a name it is. Perhaps something like `exppat`?

Reply:
ok, sure